### ISSUE TITLE
Add example for config

### ISSUE THREAD

**Issue by madsem** [ikcb/animated-tailwindcss#27]
I just installed your configuration, it seems to be great, but I cannot use it lol.
Maybe I need some more coffee, but to me this makes no sense:

```js
const withAnimations = require('animated-tailwindcss');

module.exports = withAnimations({
  // ... your config here ...
});
```

What config goes there?

**Comment by madsem** [comment]
Okay, it was that I needed more coffee.... ;) haha, sorry for the confusion.
I understand it now, it is **optional** config, it just works right out of the box after install, I just had to use the custom class names.

Thanks for making this, pretty cool stuff.

**Comment by brc-dd** [comment]
Thanks for the feedback. I'll try to add complete working examples and much detailed installation. Coming from Next.js background, it appeared to me quite clear, but the case may not be same for everyone.

Just to clarify, when you install tailwind (and initialize it using `npx tailwindcss init`), it will generate a file `tailwind.config.js` with the following content:

```js
// https://tailwindcss.com/docs/installation#create-your-configuration-file

// tailwind.config.js
module.exports = {
  purge: [],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {},
  },
  variants: {},
  plugins: [],
}
```

Now a minimal setup to use `animated-tailwindcss` will look something like this:

```js
// tailwind.config.js

const withAnimations = require('animated-tailwindcss');

module.exports = withAnimations({
  purge: [],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {},
  },
  variants: {},
  plugins: [],
})
```

The comment `// ... your config here ...` basically meant that you have to wrap *__your existing Tailwind configuration__* with `withAnimations`.

You can try this on Tailwind playground: https://play.tailwindcss.com/vLSZhGUMTz

Also, thanks for using the configuration. And indirectly pointing a bug -- actually you can't pass an empty object (or simply pass nothing) for now. Currently, at least `withAnimations({plugins: []})` is needed. I'll fix this in the next patch release.

**Comment by brc-dd** [comment]
Fixed the bug in [v2.2.1](https://www.npmjs.com/package/animated-tailwindcss/v/2.2.1). The below configuration is now the minimal configuration to use this package:

```js
// tailwind..config.js

module.exports = require('animated-tailwindcss')()
```

**Comment by madsem** [comment]
Thanks for the explanation, I actually still wasn't using it right :)

Now with this, everything is working:
```js
// tailwind.config.js

const withAnimations = require('animated-tailwindcss');

module.exports = withAnimations({
  purge: [],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {},
  },
  variants: {},
  plugins: [],
})
```
